us-medical-insurance-costs-checkpoint.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Display IQR, Standard Deviation and Outliers for each region
</pre><p>This should be changed to make use of the new modular class method, just like in the bmi cost analysis part further down.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laurenashley/US-Medical-Insurance-Costs/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Sounds good, will do</p>

Reply:
<p>Lookign into this further, we are expecting tuples from a dict being sent to analyse_costs but regions were structured differently making each region contain many single cost vals in the dict. We are however already making use of the class helper methods. </p>

Reply:
So does that mean I need to define the various dictionaries differently when making use of the methods, or have you already addressed that? (Sorry I'm a bit confused)